Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html

```diff
@@ -1,145 +1,114 @@
\ No newline at end of file
+Assignment Part One: Object-Property Pair contradictions.
+
+"PROVIDE OWL-2 DIRECT SEMANTIC BASED EXPLANATIONS
+FOR UNSATISFIABILITY AND INCONSISTENCY RESULTS
+STEMMING FROM ROLE CONSTRAINT PAIRS."
+
+1.) FUNCTIONAL COMBINED WITH TRANSITIVE BECOMES UNDECIDEABLE
+
+Functional object property denotes cardinality of (1) such relationship between 2 instances.
+For example, John (x) is connected to Mary (y) by object-property "has father" is unique.
+That is, Steve (y) can only bear the object-property relationship "has_father" with one other individual, namely, John (x).
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+When running a reasoner the combination of these becomes undecideable because a functional property must only be only be obtained between one individual and another,
+however, if it is applied transitively it whill carry over to subsequent pairs who are also connected by this same object property.
+
+
+2.) INVERSE-FUNCTIONALITY AND TRANSITIVITY BECOMES UNDECIDEABLE
+
+Inverse functionality provides that each individual paired with another with this relation can have at most one incoming connection of said object property.
+Ex: Inverse_Functional_Property "father of";
+    Father_of (Joey, Richard)      = "Joey is father_of Richard"
+--> This means that the instance, Richard, can only be inverse-fucntionally related to Joey and no other instance because any person can only have one father.
+--> Conversely the object-property "father_of" may be obtained between Joey and other instances, who will only have this relation which Joey. 
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+When running a reasoner the combination of these becomes undecideable because a functional property must only be only be obtained between one individual and another,
```

Comment:
I would rephrase this: an inverse functional property ensures that an object can be connected to at most one subject through a given object property